src/styles/addBooks.css

```diff
@@ -0,0 +1,73 @@
+form {
+  width: 90%;
+  margin-inline: auto;
+  margin-bottom: 30px;
+}
```

Comment:
- [ ] When I notice your form then it is not covering the whole width but if you see the Figma design then the input form is covering the maximum length. Kindly adjust it.
- [ ] kindly change the color of `Add buttons` as well
### Your design
![your](https://user-images.githubusercontent.com/81727988/180431309-eccd71ea-6518-4d25-b995-b5f128f3418e.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/180431344-22b3987a-83b9-47d0-8129-bea69d423377.png)